changelogs/fragments/community_vmware-dev_guide.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - community.vmware - Migrated the Developer Guide to the `community.vmware` repo (https://github.com/ansible/ansible/pull/77723).
```

Comment:
Docs only changes usually have no changelog fragment.

Reply:
> Docs only changes usually have no changelog fragment.

OK, I've removed it.